content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/artifactory/artifactory-packages-pull-through/index.md

```diff
@@ -81,22 +121,19 @@ export ARTIFACTORY_HOSTNAME=my-artifactory-hostname
 If you aren't sure of these values, you can find them in the command from the **Set Up An Alpine Client** window where you found the token:
 
 ```shell
-sudo sh -c "echo 'https://linky:<TOKEN>@example-hostname.jfrog.io/artifactory/cg-wolfi/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
+sudo sh -c "echo 'https://linky:<TOKEN>@example-hostname.jfrog.io/artifactory/cg-private/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
 ```
 
 In this example, the Artifactory user profile is `linky` and the hostname is `example-hostname`.
 
-If your Artifactory user profile is an email address, you will encounter an error unless you percent-encode the `@` sign, like this:
+> **Note**: If your Artifactory user profile is an email address, you will encounter an error unless you percent-encode the `@` sign, as in `export ARTIFACTORY_USER_PROFILE=linky%40example.com`. Here, the Artifactory user profile is `linky@example.com`, but it must be entered into the environment variable as `linky%40example.com`. 
 
-```shell
-export ARTIFACTORY_USER_PROFILE=linky%40example.com
-```
 
-Here, the Artifactory user profile is `linky@example.com`.
+## Testing Pull-Through from Private APK Repository
 
-### Testing pull through from remote package repositories interactively
+In order to test that you're able to pull packages from the remote Artifactory repository you just created, this section outlines how to build a container image (using a Chainguard image as a base) and configure it to pull packages from these repositories.
 
-In order to test that you're able to pull packages from the remote Artifactory repositories you just created, this section outlines how to build a container image (using a Chainguard image as a base) and configure it to pull packages from these repositories.
+### Testing package pull-through interactively
```

Comment:
Title case?

Reply:
Also .. I think this whole section is redundant .. the next section where the apk install is in the dockerfile is good enough and you could add a verbose flag or so if you really want.. this should not be a docker tutorial